app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -238,6 +240,11 @@ object TelemetryWrapper {
     }
 
     fun homeTileClickEvent(tile: HomeTile) {
+        if (tile.idToString() == YOUTUBE_TILE_ID) {
+            TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.HOME_TILE,
+                    Value.YOUTUBE_TILE).queue()
+        }
+
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.HOME_TILE,
```

Comment:
I'm not sure why it was originally designed this way.  The second option would be problematic, however, because it would make home tile usage statistics internally inconsistent before and after the change.

Reply:
@Baron-Severin I saw your message in the FFES channel: I don't know if Ashley (or Megan) is aware of the potential data-skewing flaws in the current implementation so I think it'd be good to check with them specifically about that before we ship this. e.g. Megan has to update any queries related to tile clicks now to remove the redundant youtube clicks.